pkg/kubectl/describe.go

```diff
@@ -1588,6 +1588,9 @@ func (dd *DeploymentDescriber) Describe(namespace, name string) (string, error)
 		fmt.Fprintf(out, "Labels:\t%s\n", labels.FormatLabels(d.Labels))
 		fmt.Fprintf(out, "Selector:\t%s\n", labels.FormatLabels(d.Spec.Selector))
 		fmt.Fprintf(out, "Replicas:\t%d updated / %d total\n", d.Status.UpdatedReplicas, d.Spec.Replicas)
+		if conditionsLen := len(d.Status.Conditions); conditionsLen > 0 {
+			fmt.Fprintf(out, "Status:\t%s\n", d.Status.Conditions[conditionsLen-1].Type)
```

Comment:
Why print the last? Seems like it makes sense to print the one that's true.

We should also display this as a column in kubectl get deployment.

I'm opposed to new phases, but I'm not opposed to computing a single Reason that summarizes all the conditions: https://github.com/kubernetes/kubernetes/issues/12143#issuecomment-160728358